.github/workflows/pr.yml

```diff
@@ -22,10 +22,10 @@ jobs:
         steps:
             - uses: actions/checkout@master
             - name: Setup PHP, with composer and extensions
-              uses: shivammathur/setup-php@master
+              uses: shivammathur/setup-php@v2
               with:
                   php-version: 7.2
-                  extension-csv: ctype, dom, iconv, json
+                  extension: ctype, dom, iconv, json
```

Comment:
This should be `extensions` (cf. https://github.com/shivammathur/setup-php#heavy_plus_sign-php-extension-support)